src/Symfony/Component/Form/Extension/Validator/Constraints/FormValidator.php

```diff
@@ -29,10 +29,7 @@ class FormValidator extends ConstraintValidator
      */
     private \SplObjectStorage $resolvedGroups;
 
-    /**
-     * @return void
-     */
-    public function validate(mixed $form, Constraint $formConstraint)
+    public function validate(mixed $form, Constraint $formConstraint): void
```

Comment:
isn't this a BC break as this class is not final ?

Reply:
yep, they all are. I'm iterating on the topic to figure out the native types we should revert on v7 (like this one, see 7a8a998f82) or the ones we can change on v6 because nobody can reasonably override the patched methods in a child class.